### PR TITLE
#41-Create models to work with MongoDB

### LINKED ISSUE #41: Create models to work with MongoDB
1. Install mongoengine
2. Write models to work with MongoDB by this doc
https://docs.google.com/document/d/197U0UH70QTsjsg9tHXBZ4rLCp39dhYKUMtnWYXW5Szo/edit?pli=1

### PR DESCRIPTION
**Pull request by Aarrtteemm123**
Create models to work with MongoDB

### CODE REVIEW

File: admin/models.py [hunk 1]
```diff
@@ -1,10 +1,75 @@
...
+from mongoengine import DateTimeField, Document, EmailField, EmbeddedDocument, EmbeddedDocumentField, FloatField, \
+    IntField, ListField, ReferenceField, StringField
 
 
-class Users(Document):
-    firstname = StringField(max_length=50, required=True)
-    lastname = StringField(max_length=50, required=True)
+class Status:
+    ACTIVE = 'active'
+    DELETED = 'deleted'
```
Comment (Palusha) [inline admin/models.py]: @Aarrtteemm123 I think u need to change `DELETED = 'deleted'` to `INACTIVE = 'inactive'`, like it's up in the code

**Review by ivan-dubei — APPROVED** [review]

File: admin/app.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def create_user():
     form = AddUserForm(request.form)
     if request.method == 'POST' and form.validate_on_submit():
         try:
-            user = Users(email=form.email.data)
+            user = User(email=form.email.data)
             user.firstname = form.firstname.data
             user.lastname = form.lastname.data
```
Comment (Palusha) [inline admin/app.py:49]: @Aarrtteemm123 Add `user.login = form.login.data`

File: admin/app.py [hunk 3]
```diff
@@ -63,7 +63,7 @@ def create_user():
 @app.route('/update_user/<string:_id>', methods=['POST', 'GET'])
 def update_user(_id: str):
     try:
-        user = Users.objects.get(id=ObjectId(_id))
+        user = User.objects.get(id=ObjectId(_id))
         form = UpdateUserForm(request.form, role=user.role, status=user.status)
         if request.method == 'POST' and form.validate_on_submit():
```
Comment (Palusha) [inline admin/app.py:68]: @Aarrtteemm123 Add `user.login = form.login.data`

File: admin/models.py [hunk 4]
```diff
@@ -1,10 +1,75 @@
...
+
+
+class User(Document):
+    firstname = StringField(max_length=100, min_length=1, required=True)
+    lastname = StringField(max_length=100, min_length=1, required=True)
     email = EmailField(required=True, unique=True)
-    password = StringField(required=True)
-    role = StringField(required=True)
-    status = StringField(required=True, default='active')
+    login = StringField(required=True)
```
Comment (Palusha) [inline admin/models.py]: @Aarrtteemm123 Add `unique=True` to field "**login**"

**Review by Palusha — APPROVED** [review]

**Pull request merged**